pages/linux/limactl.md

```diff
@@ -0,0 +1,29 @@
+# limactl
+
+> Virtual machine manager for Linux guests, with multiple VM templates available.
+> Can be used to run containers on macOS, but also for generic virtual machine use cases on macOS and Linux hosts.
+> More information: <https://github.com/lima-vm/lima>.
+
+- Pick any of the available distributions, see templates in `/usr/local/share/lima/templates/`:
+
+`LIMA_DISTRO={{debian|fedora|ubuntu|…}}`
+
+- Create a virtual machine using the default settings (this will download the VM image on the first run):
+
+`cat /usr/local/share/lima/templates/$LIMA_DISTRO.yaml | limactl create --name=$LIMA_DISTRO-vm -`
+
+- Start the VM instance (this might install some dependencies in the VM and take a few minutes):
+
+`limactl start $LIMA_DISTRO-vm`
+
+- Open a remote shell inside the VM:
+
+`limactl shell $LIMA_DISTRO-vm`
+
+- Stop/shutdown the VM instance:
+
+`limactl stop $LIMA_DISTRO-vm`
+
+- Delete the VM instance:
+
+`limactl remove $LIMA_DISTRO-vm`
```

Comment:
```suggestion
- List VM instances:

`limactl list`

- Start the VM instance (this might install some dependencies in the VM and take a few minutes):

`limactl start {{vm_name}}`

- Open a remote shell inside the VM:

`limactl shell {{vm_name}}`

- Run a command inside the VM:

`limactl shell {{vm_name}} {{command}`

- Stop/shutdown the VM instance:

`limactl stop {{vm_name}}`

- Delete the VM instance:

`limactl remove {{vm_name}}`
```

Reply:
Hm.. I see this is opinionated, but I think that using the environment variable for selecting the distro and using that in the vm name is a neat trick that I find useful and wanted to share here. In my opinion, this adds additional value to the cheatsheet compared to the output of `limactl --help` (which I found a bit suboptimal which was my motivation to open this pr).

Reply:
In my opinion, the great thing about tldr pages is that it gives me a quick start with clis often when I am fine with opinionated examples.. for cases where this is not suitable for my needs I need to do further research.